lib/dry/validation/result.rb

```diff
@@ -81,3 +81,11 @@ def result_ast
     end
   end
 end
+
+begin
```

Comment:
ah, I see, it will work differently for dry-struct and dry-monads. I mean as long as dry-t depends on dry-m, we won't have LoadError. However, dry-struct will be required in specs only, so we won't try to require it in dry-v, leaving that step for the user, right (by calling `Dry::Validation.load_extensions(:struct)`)?